src/hooks/use-autofill-height.js

```diff
@@ -8,14 +8,20 @@ const getNodeFullHeight = node => {
       getComputedStyle(node.nextSibling).marginTop
     );
   }
-  return (
+  const height =
     node.offsetHeight +
     parseFloat(style.marginTop) -
     nextSiblingMarginTop +
-    parseFloat(style.marginBottom)
-  );
+    parseFloat(style.marginBottom);
+  return height;
 };
 
+const isCurrentNodeAutoFill = current =>
+  current.classList.contains('spectacle-auto-height-fill') ||
+  (current.tagName.toLowerCase().includes('pre') &&
+    current.childNodes &&
+    current.childNodes[0].classList.contains('spectacle-auto-height-fill'));
```

Comment:
mdx/react adds a second `pre` tag as a child, so we need to account for that when allowing for auto-height components.